packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/forceFunctionCreate.test.ts

```diff
@@ -124,6 +125,17 @@ describe('Force Function Create', () => {
       );
     });
 
+    it('Log metric should add additional language property for telemetry', async () => {
+      const telemetryServiceStub = stub(telemetryService, 'sendCommandEvent');
```

Comment:
Looks like the CircleCI check is failing because this stub needs to be restored! I would declare this variable at the top of the describe() block and then restore it in the afterEach(). Then the checks will likely pass

Reply:
Thanks @maxcamp!